workflows/raredisease.nf

```diff
@@ -413,7 +414,7 @@ workflow RAREDISEASE {
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-    if (!params.skip_repeat_calling && params.analysis_type.equals("wgs") ) {
+    if (!(params.skip_subworkflows && params.skip_subworkflows.split(',').contains('repeat_calling')) && params.analysis_type.equals("wgs") ) {
```

Comment:
why repeat expansion is only possible for wgs? might not be ideal but can be run with wes

Reply:
Tools used here have been a bit flaky with wes samples. I could add an override in future, like I do with "run_mt_for_wes" parameter so users can choose to turn on this workflow for wes samples. Will do that in a different PR.